lib/stdio/async.js

```diff
@@ -88,10 +88,9 @@ const setStandardStreamMaxListeners = (stream, {signal}) => {
 };
 
 // `source.pipe(destination)` adds at most 1 listener for each event.
-// We also listen for the stream's completion using `finished()`, which adds 1 more listener.
 // If `stdin` option is an array, the values might be combined with `merge-streams`.
 // That library also listens for `source` end, which adds 1 more listener.
-const maxListenersIncrement = 3;
```

Comment:
Value changed since we are not calling `finished(stream)` on `process.std*` anymore.